pkg/administration/events/singleton.go

```diff
@@ -11,7 +11,7 @@ var (
 	stream Stream
 )
 
-// Singleton returns an instance of the notification stream.
+// Singleton returns an instance of the administration events stream.
 func Singleton() Stream {
```

Comment:
Generally I would discourage the use of singletons in package. There can be weird side effects if called in different places that expect new instances

Reply:
Discussed this offline, I've added a description to make it more clear as to why this is required / can be done.